### PR TITLE
Optimize Bintray and JCenter for the dependencies

### LINKED ISSUE #328: Remove usages of `jcenter()`
## Kind (Required)

- Improvement

 ## Overview (Required)
Warning by Android Studio

```
Please remove usages of `jcenter()` Maven repository from your build scripts and migrate your build to other Maven repositories.
This repository is deprecated and it will be shut down in the future.
See http://developer.android.com/r/tools/jcenter-end-of-service for more information.
Currently detected usages in: root project 'conference-app-2021', project ':android', project ':data', ...
Affected Modules: android
```

### PR DESCRIPTION
**Pull request by ykws**
## Issue
- close #328

## Overview (Required)
- **Not completed to remove JCenter**
- Remove Bintray
- JCenter only for the dependencies described within the block

## Links
- [Into the Sunset on May 1st: Bintray, JCenter, GoCenter, and ChartCenter](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/)
- Refs snippet: https://github.com/Kotlin/kotlinx.html/issues/173#issuecomment-800504909
- Inquiry this snippet: https://github.com/Kotlin/kotlinx.collections.immutable/issues/96#issuecomment-801299174

## Screenshot
Screen not changed.

### CODE REVIEW

**Comment by anandwana001** [comment]
@ykws Is it building successfully on the local machine?

reference article - https://jeroenmols.com/blog/2021/02/04/migratingjcenter/

**Comment by ykws** [comment]
I'm sorry. I have abandoned the perfect fixes for this issue.
Ready for review this pull request as a better practice that now I think.

**Comment by punchdrunker** [comment]
👀

File: build.gradle [hunk 1]
```diff
@@ -8,8 +8,15 @@ buildscript {
     repositories {
         google()
         mavenCentral()
-        jcenter()
-        maven { url = "https://dl.bintray.com/kotlin/kotlin-eap" }
+        // This snippet enables jcenter only for the dependencies described within the block.
```
Comment (punchdrunker) [inline build.gradle:11]: thank you for the helpful comment

**Review by punchdrunker — APPROVED** [review]
💯

**Pull request merged**